### PR TITLE
Reference EIP-161 for the 20 million dust account deletion

### PR DESCRIPTION
**Pull request by pyskell**
As noted in issue #477

### CODE REVIEW

File: clients.asciidoc [hunk 1]
```diff
@@ -562,3 +562,7 @@ https://MyCrypto.com
 Mist is the first ever Ethereum enabled browser, built by the Ethereum Foundation. It also contains a browser-based wallet that was the first ever implementation of the ERC20 token standard (Fabian Vogelsteller, author of ERC20 was also the main developer in Mist). Mist was also the first wallet to introduce the camelCase checksum (EIP-155, see <<eip-155>>). Mist runs a full node, and offers a full DApp browser with support for Swarm based storage and ENS addresses. Find it at:
 
 https://github.com/ethereum/mist
+
+[bibliography]
+=== References
+- [[[1]]] EIP-161: http://eips.ethereum.org/EIPS/eip-161
```
Comment (fjrojasgarcia) [inline clients.asciidoc:568]: Sorry, Are you trying to use an inline anchor?

_Defining an inline anchor_
`[[bookmark-a]]Inline anchors make arbitrary content referenceable.`

[27.2. Defining an Anchor - asciidoctor.org](https://asciidoctor.org/docs/user-manual/#anchordef)

Comment (pyskell) [reply clients.asciidoc:568]: Yes, an inline anchor to "[1]".

Comment (fjrojasgarcia) [reply clients.asciidoc:568]: Go it! Thanks!

File: clients.asciidoc [hunk 2]
```diff
@@ -454,7 +454,7 @@ While it is possible to fully sync the blockchain this way, the sync will take a
...
+For example, on Ethereum, a new client will make rapid progress until it reaches block 2,283,397. This block was mined on September 18th, 2016 and marks the beginning of the DoS attacks. From this block and until block 2,700,031 (November 26th, 2016), the validation of transactions becomes extremely slow, memory intensive, and I/O intensive. This results in validation times exceeding 1 minute per block. Ethereum implemented a series of upgrades, using hard forks, to address the underlying vulnerabilities that were exploited in the denial of service. These upgrades also cleaned up the blockchain by removing some 20 million empty accounts created by spam transactions. <<[1]>>
```
Comment (lrettig) [inline clients.asciidoc:457]: > These upgrades also cleaned up the blockchain by removing some 20 million empty accounts created by spam transactions.

This is technically slightly inaccurate (see my comment here: https://github.com/ethereumbook/ethereumbook/issues/477#issuecomment-383556405). I might reword this as, "by allowing the removal of...".

**Pull request merged**